docs/qa/test_engineering

```diff
@@ -0,0 +1,44 @@
+Normandy Test Engineering
+=========================
+This is the repository for all information related to test engineering for the Normandy project.  This will outline our resources and test process for releases.
+
+The primary focus of testing will be on the Firefox client and the API server. The Normandy Firefox client follows the release process and train schedule for the Firefox browser. The Normandy server is tested using the API server and recipes.
+
+The Delivery Console is not currently being tested, as it’s a much lower priority. 
+
+The test strategy incorporates manual and automated testing. Different approaches are outlined for each area. 
+
+Normandy Server
+--------------
+The Normandy Server is an API server with recipes. This feature is high priority, and needs to maintain a high level of service. There are two versions with different functionality. V1 is the server clients connect to. V3 is the server that Admins connect to.
+
+- v1
+Automation:
+Test Engineer: chartjes
+Status:  In process
+
+- v3
+Automation:
+Test Engineer: b4hand
+Manual tests: none
+Status: In process
+
+Normandy Firefox client
+-----------------------
+     The Normandy Firefox client queries the Normandy server for instructions and picks up recipe changes. 
+QA: cmuresan
+Automation: https://github.com/18epedersen/Normandy-e2e-tests Status: outdated
```

Comment:
This isn't correct. That test suite was for the predecessor to Delivery Console, not for the Normandy Firefox client.